package.json

```diff
@@ -27,13 +28,14 @@
     "ember-auto-import": "^1.5.3",
     "ember-cli-babel": "^7.18.0",
     "ember-cli-htmlbars": "^4.2.3",
-    "xstate": "^4.8.0"
+    "ember-resource": "https://github.com/pzuraq/ember-usable#0d03a50",
+    "xstate": "^4.8.0",
+    "@glimmer/tracking": "^1.0.0"
   },
   "devDependencies": {
     "@ember/optional-features": "^1.3.0",
     "@ember/render-modifiers": "^1.0.2",
     "@glimmer/component": "^1.0.0",
-    "@glimmer/tracking": "^1.0.0",
```

Comment:
Is moving this to `dependencies` this intended?

Reply:
As long as we're good with dropping support for pre 3.15, I'd assume so ;)

Reply:
I think I moved it because we use it in the usable. 🤔 will recheck